conf/app.routes

```diff
@@ -10,6 +10,12 @@ PUT           /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn               @uk
 GET           /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn               @uk.gov.hmrc.agentclientrelationships.controllers.RelationshipsController.checkWithMtdVat(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn, vrn: uk.gov.hmrc.agentmtdidentifiers.model.Vrn)
 DELETE        /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn               @uk.gov.hmrc.agentclientrelationships.controllers.RelationshipsController.deleteVatRelationship(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn, vrn: uk.gov.hmrc.agentmtdidentifiers.model.Vrn)
 
+
+#NI-ORG relationship
+PUT           /agent/:arn/service/HMRC-NI-ORG/client/EORI/:eori             @uk.gov.hmrc.agentclientrelationships.controllers.RelationshipsController.createForNiOrg(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn, eori: uk.gov.hmrc.agentmtdidentifiers.model.Eori)
```

Comment:
yes, all the PUTS above are not doing any service specific logic , so can remove `service/HMRC-NI-ORG` from the urls ?

Reply:
I would prefer to keep it as is, for now, we can discuss it next week as team